frontend/src/pages/Nodes/NodesTable.js

```diff
@@ -29,11 +29,12 @@ const groupBy = nodes => {
     if (node.currentAccess.accessType !== "none") {
       const key = node.address.organization;
       const index = acc.findIndex(x => x.organization === key);
+      const permissionsString = `${node.currentAccess.accessType} (${node.isConnected ? strings.status.connection : strings.status.not_connected})`;
```

Comment:
Might be a better idea to show the connection status not as a string but as a green/red dot, like in the status page (`frontend/src/pages/Status/StatusTable`)

Reply:
@georgimld, @Stezido text representation is simple and do the thing, of course the table would look more perky with circles but displaying multiple circles on the same row would look "dirty" with greater count of nodes. I propose to put each node on separated node and create new column Connection to look like in Status table then. What do you think?

Reply:
@m-janos I also think it would be a good idea to display each node on a separate row. We already discussed this in yesterday's call with @Stezido and @SamuelPull and it was suggested for the "last seen" issue, right?

Reply:
maybe like this + the "last seen" timestamp in another column eventually?

![Screenshot 2021-02-18 at 14 42 41](https://user-images.githubusercontent.com/48566702/108365598-ad1aaf00-71f7-11eb-8283-280414f7b5b3.png)

Reply:
@georgimld all right, this will be resolved in #774 and #785